lib/modules/manager/terraform/lockfile/hash.spec.ts

```diff
@@ -125,4 +140,196 @@ describe('modules/manager/terraform/lockfile/hash', () => {
       'h1:I2F2atKZqKEOYk1tTLe15Llf9rVqxz48ZL1eZB9g8zM=',
     ]);
   });
+
+  it('full walkthrough on terraform cloud', async () => {
```

Comment:
Update full walk through to be still one

Reply:
I left this test in, since it is testing something different from the 'full walkthrough'. That test is testing for `releases.hashicorp.com` endpoint providers, while this one is for the `registry,terraform.io` and alike. This is also where the ziphashes are available, not on the other endpoint. I have reduced the complexity slightly and changed it to the hashicorp/google provider to also include the manifest.json test in this one.